tkg/client/common.go

```diff
@@ -364,11 +365,14 @@ func (c *TkgClient) ShouldDeployClusterClassBasedCluster(isManagementCluster boo
 		return true, nil
 	}
 
+	// Ignore the checksum check if config variable 'SUPPRESS_PROVIDERS_UPDATE' is set.
+	isSuppressProvidersUpdateSet := os.Getenv(constants.SuppressProvidersUpdate) != ""
```

Comment:
May I ask why the `SUPPRESS_PROVIDERS_UPDATE` was required to set to number `1` instead of boolean or other values? Does this number contain any meaning?

Reply:
@12345lcr , no, `SUPPRESS_PROVIDERS_UPDATE` can be set to any string. As long as it is not an empty string, then `isSuppressProvidersUpdateSet` will be `true`.